operator/api/redpanda/v1alpha2/redpanda_clusterspec_types.go

```diff
@@ -256,16 +256,21 @@ type RedpandaConsole struct {
 	// Specifies whether a Deployment should be created for Redpanda Console.
```

Comment:
Please update the comment on this struct to indicate that it now represents the union of `console.PartialValues` and `consolev3.PartialValues`.

Could you also file a JIRA to follow up on handling the migration of console v2 to v3 in the operator?

Reply:
ok